packages/mdc-checkbox/index.js

```diff
@@ -34,12 +34,12 @@ export class MDCCheckbox extends MDCComponent {
   }
 
   /**
-   * @return {InputElementState|undefined}
+   * @return {SelectionControlState|undefined}
    * @private
    */
   get nativeCb_() {
     const {NATIVE_CONTROL_SELECTOR} = MDCCheckboxFoundation.strings;
-    const cbEl = /** @type {InputElementState|undefined} */ (
+    const cbEl = /** @type {SelectionControlState|undefined} */ (
```

Comment:
`{SelectionControlState|undefined}` -> `{?SelectionControlState}`

`querySelector()` returns `null`, not `undefined`.